Add criteria to tournaments closes #1118

timothyarmes opened the pull request.

timothyarmes committed:
Add criteria to tournaments closes #1118

timothyarmes committed:
Translations

pascalaubry commented on RELEASE_NOTES.md:
```diff
@@ -9,6 +9,10 @@ English version below
 
+## Tournois
+
+- Les tournois peuvent désormais avoir des critères de joueur·euse·s associés (3.1.0) ✅
```
```suggestion
- Les tournois peuvent désormais avoir des critères de joueur·euses associés (3.1.0)
```

pascalaubry commented on locale/en/LC_MESSAGES/messages.po:
```diff
@@ -6397,3 +6418,6 @@ msgstr ""
 
+#~ msgid "All tournaments"
+#~ msgstr ""
+
```
```suggestion
```

pascalaubry commented on locale/fr/LC_MESSAGES/messages.po:
```diff
@@ -73,6 +73,12 @@ msgid_plural "%(num)d players did not check-in"
+msgid "%(num)d player does not match criteria."
+msgid_plural "%(num)d players do not match criteria."
+msgstr[0] "%(num)d joueur·euse ne correspond pas aux critères."
+msgstr[1] "%(num)d joueur·euse·s ne correspondent pas aux critères."
```
```suggestion
msgstr[1] "%(num)d joueur·euses ne correspondent pas aux critères."
```

pascalaubry commented on locale/fr/LC_MESSAGES/messages.po:
```diff
@@ -1417,6 +1423,9 @@ msgstr "Définir comme public"
 msgstr "Définir les critères selon lesquels les joueur·euses appartiennent à cette catégorie."
 
+msgid "Define the criteria for players to participate in this tournament."
+msgstr "Définissez les critères permettant aux joueur·euse·s de participer à ce tournoi."
```
```suggestion
msgstr "Définissez les critères permettant aux joueur·euses de participer à ce tournoi."
```

pascalaubry commented on src/database/sql_server/sql_server.py:
```diff
@@ -70,7 +70,7 @@ def dump(
     """Base class for SQL-server databases using python-tds."""
 
-    DEFAULT_TIMEOUT: int = 3
+    DEFAULT_TIMEOUT: int = 10
```
Was that needed for slow networks?

timothyarmes replied:
Yes, it's needed with the complex search query

pascalaubry reviewed (dismissed):
Details only.

timothyarmes committed:
Update RELEASE_NOTES.md

Co-authored-by: Pascal AUBRY <66429491+pascalaubry@users.noreply.github.com>
Update locale/en/LC_MESSAGES/messages.po

Co-authored-by: Pascal AUBRY <66429491+pascalaubry@users.noreply.github.com>
Update locale/fr/LC_MESSAGES/messages.po

Co-authored-by: Pascal AUBRY <66429491+pascalaubry@users.noreply.github.com>
Update locale/fr/LC_MESSAGES/messages.po

Co-authored-by: Pascal AUBRY <66429491+pascalaubry@users.noreply.github.com>

pascalaubry committed:
Merge branch 'dev' into feature/critiria
